src/seedu/addressbook/Main.java

```diff
@@ -84,10 +84,13 @@ private void runCommandLoopUntilExitCommand() {
         do {
             String userCommandText = ui.getUserCommand();
             command = new Parser().parseCommand(userCommandText);
-            CommandResult result = executeCommand(command);
-            recordResult(result);
-            ui.showResultToUser(result);
-
+            try {
+                CommandResult result = executeCommand(command);
+                recordResult(result);
+                ui.showResultToUser(result);
+            } catch (StorageOperationException e) {
+                ui.showToUser(e.getMessage());
```

Comment:
A few things to note wrt exceptions:
- In general, it is better to handle exceptions closer to where they occur
- Printing the error message alone is not the considered "good" exception handling. Think what better can be done to improve user experience?

BTW, this commit doesn't belong to the Class LO. 
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later.